examples/device/src/default_responder.c

```diff
@@ -42,7 +42,7 @@ static const uint16_t kSupportedPIDList[NUM_SUPPORTED_PIDS] = {
 static const uint8_t kDeviceInfo[] = {
     0x01, 0x00, /* RDM Protocol version */
     0xe1, 0x33, /* Device Model ID */
-    0xe1, 0x33, /* Product Category */
+    0x71, 0x01, /* Product Category */
```

Comment:
I guess this would be nice if this used the E120_PRODUCT_CATEGORY_TEST_EQUIPMENT @samkearney , but given you're building a raw byte array and as far as I'm aware you don't have an easy thing to split a uint16 into two bytes I haven't bothered.